common/src/stack/command/stack/commands/add/host/interface/alias/__init__.py

```diff
@@ -56,27 +56,23 @@ def run(self, params, args):
 		except socket.error:
 			pass
 
-		for dict in self.call('list.host.alias'):
-			if (
-				alias == dict['alias'] and
-				interface == dict['interface'] and
-			 	host == dict['host']
+		for row in self.call('list.host.interface.alias'):
+			if alias == row['alias'] and (
```

Comment:
I spent too long staring at this conditional trying to figure out what it wants to prevent and what it wants to allow. Consider adding a comment describing it's purpose.